uap_tracker/no_op_visualiser.py

```diff
@@ -0,0 +1,7 @@
+from visualizer import Visualizer
+
+
+class NoOpVisualiser(Visualizer):
```

Comment:
Not sure I'm understanding the utility of this class. Why doesn't using 'none' as the visualizer achieve the same thing?

Reply:
I removed the none visualiser and replaced it with noop as I did not like the none name. It was still a visualiser with a name of none, I thought it would be confusing. More a naming thing than anything else.

Reply:
I see, you removed the DefaultVisualizer, and replaced it with NoOp. But there's also a 'none' visualizer which doesn't even need a NoOp, So i'd say just drop the NoOp

Reply:
After discussion with Mike, NoOp is inheriting an info message from parent, which is why it's needed.